src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1064,7 +1082,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IConversionExpression_IsExplicit As Boolean Implements IConversionExpression.IsExplicit
+        Private ReadOnly Property IConversionExpression_Conversion As IConversion Implements IConversionExpression.Conversion
+            Get
+                Return New Conversion(ConversionKind.UserDefined, Me.Call.Method)
```

Comment:
BoundUserDefinedConversion is not supposed to implement IConversionExpression, see #8872.